components/PlantCard/index.js

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image"
+import styled from "styled-components"
+
+export default function PlantCard({ image, name, botanicalName }) {
+
+    return (
+        <section>
```

Comment:
an article would be more fitting here, sections are for entire parts of the web page.